src/mono/mono/metadata/sgen-mono.c

```diff
@@ -2897,6 +2897,7 @@ sgen_client_binary_protocol_collection_end (int minor_gc_count, int generation,
 void
 sgen_client_schedule_background_job (void (*cb)(void))
 {
+	// TODO: is `cb` wrapped with MONO_ENTER_GC_UNSAFE/MONO_EXIT_GC_UNSAFE ?
```

Comment:
@lambdageek do I need to handle this boundary for wasm ? Or is that callback ok to call from outside as is?